### PR TITLE
Layout ng9

### PR DESCRIPTION
**Pull request by raitisbe**

### CODE REVIEW

File: components/layout/layout.component.ts [hunk 1]
```diff
@@ -41,12 +41,6 @@ export class HsLayoutComponent {
...
       innerElement: '.hs-map-container',
     });
-    if (HsConfig.sidebarPosition === 'left') {
-      this.HsLayoutService.contentWrapper.classList.add('flex-reverse');
-      this.HsLayoutService.sidebarRight = false;
-    } else if (HsConfig.sidebarPosition != 'invisible') {
-      HsConfig.sidebarPosition = 'right';
-    }
 
     if (this.HsConfig.importCss == undefined || this.HsConfig.importCss) {
```
Comment (FilipLeitner) [inline components/layout/layout.component.ts:45]: What would be the use case when we dont want to import css?

File: components/layout/layout.service.ts [hunk 2]
```diff
@@ -1,197 +1,191 @@
...
-        },
-        {
-          enabled: true,
-          order: 8,
-          title: 'Save composition',
-          description: 'Save content of map to composition',
-          name: 'saveMap',
-          directive: 'hs.save-map',
-          mdicon: 'save',
+  data = {
```
Comment (FilipLeitner) [inline components/layout/layout.service.ts:9]: This object seems to be deprecated

File: components/layout/layout.component.ts (hunk 1 above)
Comment (raitisbe) [reply components/layout/layout.component.ts:45]: if hslayers breaks some styles in the container application

File: components/layout/layout.service.ts (hunk 2 above)
Comment (raitisbe) [reply components/layout/layout.service.ts:9]: True, it was in the material design template. Deleted

**Pull request merged**